includes/Anthropic/Anthropic_AI_API_Client.php

```diff
@@ -29,20 +37,17 @@ class Anthropic_AI_API_Client extends Generic_AI_API_Client {
 	 * @param string              $default_api_version The default API version.
 	 * @param string              $api_name            The (human-readable) API name.
 	 * @param Request_Handler     $request_handler     The request handler instance.
-	 * @param Authentication|null $authentication      Optional. The authentication instance. Default null.
+	 * @param Authentication      $authentication      The authentication instance.
 	 */
 	public function __construct(
 		string $default_base_url,
 		string $default_api_version,
 		string $api_name,
 		Request_Handler $request_handler,
-		Authentication $authentication = null
+		Authentication $authentication
```

Comment:
Instead of changing the parameter to no longer be nullable (which modifies the API), shouldn't we rather fix the class property to be properly declared as nullable?
```suggestion
		?Authentication $authentication
```

Reply:
Oh awesome, thanks for the feedback! I always thought the nullable type syntax (like `?Authentication`) was a PHP 7.4 thing, but turns out it's been around since 7.1. Appreciate the nudge. I'll align the other places as well. Cheers!